### ISSUE TITLE
ValueError: not enough values to unpack (expected 2, got 1) in pdfminer/converter.py in paint_path() function in develop branch

### ISSUE THREAD

**Issue by EucliTs0** [pdfminer/pdfminer.six#729]
**ValueError: not enough values to unpack (expected 2, got 1)**

Hello,

With the current refactoring in the converter.py module (develop branch) I notice a bug when parsing specific PDF documents.

More specifically, the code block:
```
raw_pts = [cast(Point, p[-2:] if p[0] != "h" else path[0][-2:]) for p in path]
pts = [apply_matrix_pt(self.ctm, pt) for pt in raw_pts]
```
in paint_path() function introduces error because sometimes the **_path_** variable has only the value '(h, )' at index 0 and nothing else. Thus, when calling apply_matrix_pt(), when we arrive at command (x, y) = v we get the ValueError.

Traceback: 

  ```
File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/pdfinterp.py", line 1005, in process_page
    self.render_contents(page.resources, page.contents, ctm=ctm)

  File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/pdfinterp.py", line 1023, in render_contents
    self.execute(list_value(streams))

  File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/pdfinterp.py", line 1051, in execute
    func()

  File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/pdfinterp.py", line 586, in do_s
    self.do_S()

  File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/pdfinterp.py", line 578, in do_S
    self.device.paint_path(self.graphicstate, True, False, False,

  File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/converter.py", line 121, in paint_path
    pts = [apply_matrix_pt(self.ctm, pt) for pt in raw_pts]

  File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/converter.py", line 121, in <listcomp>
    pts = [apply_matrix_pt(self.ctm, pt) for pt in raw_pts]

  File "/home/dtsolakidis/.local/lib/python3.8/site-packages/pdfminer/utils.py", line 253, in apply_matrix_pt
    (x, y) = v
```

Also a snapshot of the code that is used:
```

    rsrcmgr = PDFResourceManager()

    
    laparams = LAParams(all_texts=True)

    pdf_input = io.BytesIO(pdf_file)
    output = io.BytesIO()
    
    xml_converter = XMLConverter(rsrcmgr,
                                 outfp=output,
                                 laparams=laparams,
                                 stripcontrol=True)

   
    interpreter = PDFPageInterpreter(rsrcmgr, xml_converter)

    
    for page in PDFPage.get_pages(pdf_input, maxpages=number_pages,
                                  check_extractable=False):
       interpreter.process_page(page)
```
Unfortunately, I cannot add the PDF because I deal with sensitive information documents. I am posting this bug I encounter, because it is not only in one but in many documents in the dataset. With the previous version (20201018) there are no issues at all. Perhaps, some other member have already encounter this problem.

I installed the package from pip : https://pypi.org/project/pdfminer.six/  , version 20211012

```
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.4 LTS
Release:	20.04
Codename:	focal

Python: 3.8.5
```

**Comment by Nebukadneza** [comment]
Hi,

I’m also running into this as a `paperless-ngx` user, and my sample document unfortunately also has some medium-sensitive content. I’d be comfortable with sharing the file with a interested developer personally though. I’ve joined the gitter chat as `Nebukadneza`, so please do feel free to reach!

**Comment by jsvine** [comment]
@EucliTs0 & @Nebukadneza: I cannot be certain without seeing the specific PDFs you're trying to parse, but I have filed a PR that I believe would likely fix this for you: https://github.com/pdfminer/pdfminer.six/pull/749

**Comment by Nebukadneza** [comment]
I can confirm this works after manually applying the branches patch in my paperless docker! Thanks a lot for the immensely quick and "blind" fix!

**Comment by pietermarsman** [comment]
Fixed by #749
